db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
```

Comment:
Prefer `address` as that's what we use in other tables where it could hold either ip or hostname.